generator/src/test/java/com/scottlogic/deg/generator/cucumber/features/operators/grammatical/If.feature

```diff
@@ -2482,21 +2480,19 @@ Feature: Values can be specified by using if, then and else constraints
       """
       {
         "if": { "field": "foo", "is": "equalTo", "value": 1 },
-        "then": { "field": "bar", "is": "equalTo", "value": 1 },
-        "else": { "field": "bar", "is": "granularTo", "value": 0.01 }
+        "then": { "field": "bar", "is": "equalTo", "value": 3.33 },
+        "else": { "field": "bar", "is": "granularTo", "value": 0.1 }
       }
       """
     Then the following data should be generated:
       | foo   | bar  |
-      | 1     | 1    |
-      | 1.1   | 3.33 |
-      | 1.11  | 3.33 |
-      | 1.111 | 3.33 |
+      | 1     | 3.33 |
+      | 1.1   | 2.2  |
+      | 1.11  | 2.2  |
+      | 1.111 | 2.2  |
 
-  @ignore #issue 363
   Scenario: Running an if request that contains a contradictory granularTo constraint within its if statement should be successful
     Given foo is in set:
-      | 1     |
```

Comment:
Shouldn't this be left in the set?
Otherwise we never prove that when `foo granularTo 1` bar = 1?

Reply:
I thijnk this one is supposed to be contradictory so should not contain anything that matches the if statement